manifests/machineconfigpool.crd.yaml

```diff
@@ -22,6 +22,10 @@ spec:
     description: When progress is blocked on updating one or more nodes, or the pool configuration is failing.
     name: Degraded
     type: string
+  - JSONPath: .status.machineCount
+    description: Total number of machines in the machine config pool
+    name: MachineCount
```

Comment:
Can we show "degraded", "ready", and "updated" counts as well?

Reply:
there's also `UnavailableMachineCount`, no need for it to show here?